entrypoint.sh

```diff
@@ -75,7 +77,7 @@ cd "$ROOT"
 hostWorkspacePath=`$util getWorkspacePathForVolumeMounting $LOCAL_WORKSPACE`
 
 # Export some APPSODY env vars
-export APPSODY_MOUNT_HOME=`/codewind-workspace/.extensions/appsodyExtension/scripts/get-home.sh | xargs $util getWorkspacePathForVolumeMounting`
+export APPSODY_MOUNT_HOME=`$DIR/scripts/get-home.sh | xargs $util getWorkspacePathForVolumeMounting`
 export APPSODY_MOUNT_CONTROLLER="$hostWorkspacePath/.extensions/appsodyExtension/bin/appsody-controller"
```

Comment:
The APPSODY_MOUNT_CONTROLLER should also be using $DIR

Reply:
hmmm... that is a very good catch